src/execution_plan/ops/shared/create_functions.h

```diff
@@ -24,6 +24,7 @@ typedef struct {
 	PendingProperties **node_properties;
 	PendingProperties **edge_properties;
 
+	int **node_labels;
```

Comment:
The reason for the duplication is that we have one `node_ctx` for each `CREATE` pattern, but can have N new nodes:
```
UNWIND range(0,20) as i CREATE (:L {v: i})
```
We currently have no way to backtrack from a new node to its parent context. We could add an array to do this instead of duplicating the labels array, but I don't think there would be a significant advantage to this approach.